.github/workflows/build_cygwin_gap_release.yml

```
@@ -31,11 +35,11 @@ jobs:
       run: |
         @ECHO ON
         SET CHERE_INVOKING=1
-        C:\cygwin64\bin\bash -l -c 'bash release_gap.sh ARCH=${{ matrix.arch }}'
+        C:\cygwin64\bin\bash -l -c 'bash release_gap.sh ARCH=${{ matrix.arch }} SAGE_VERSION=${{ matrix.gapbranch }}'

```

Why is `SAGE_VERSION` set to `matrix.gapbranch`? It seems at least on of these may be misnamed? If this is intentional and there are good reasons not to change it, perhaps at least a comment could explain what's going on?

UPDATE: OK, after reading more, it turns out this is because these scripts/files here are based on things coming from Sage, and there everything is of course prefixed with `SAGE_`... and it makes sense not to rename everything here, so that the diff is kept small and we can more easily rebase our changes on a new version of the sage code.

So perhaps like this:
```suggestion
        # pass the branch to the release script -- for technical and historical reasons, the variable is named SAGE_VERSION
        C:\cygwin64\bin\bash -l -c 'bash release_gap.sh ARCH=${{ matrix.arch }} SAGE_VERSION=${{ matrix.gapbranch }}'
```

---

Yes, that's right. If Chris or someone else ever tries to 'officially' unify this repo and the original sage repo, then maybe removing all the unnecessary `SAGE_` prefixes would be a good idea.

---

`#` isn't syntax for a comment in `cmd`, so I'll accept your comment but put it just above the step in the workflow.